src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,35 @@
+package codeu.chat.common;
+
+// why is this class in the util package? Can we combine the two ServerInfo classes in the
+// codeu.chat.common package?
+
+import codeu.chat.util.Time;
+import codeu.chat.util.Uuid;
+
+import java.io.IOException;
+
+public final class ServerInfo {
+
+    public Time startTime;
```

Comment:
Please list static fields separate from non-static ones, keeping static ones first.

Also, I would recommend keeping the member fields immutable and hidden:
public final Uuid version;
public final Time startTime;

This way you make the whole class immutable, making it easier to maintain/test in the future.